src/components/FieldsetHeader/FieldsetHeader.tsx

```diff
@@ -111,6 +111,13 @@ export const FieldsetHeader: FC<FieldsetHeaderProps> = ({
     const id = useMemoizedId();
     const clickOnNotDisabled = () => !disabled && onClick && onClick();
 
+    let headingFontWeightClass = 'tw-font-normal';
+    if (bold) {
+        headingFontWeightClass = 'tw-font-bold';
+    } else if (size === FieldsetHeaderSize.Medium) {
+        headingFontWeightClass = 'tw-font-medium';
+    }
+
```

Comment:
Can't we have a `Medium sized bold Fieldset Header`?
This code will apply only the bold in that case... Even if `size === FieldsetHeaderSize.Medium`

Reply:
I agree, let's discuss it with a designer. Since the goal or PR was to fight the code complexity I would go with the same logic for now.

Reply:
@fabriciomendonca please post the question to #fondue-questions-and-feedback

Reply:
Actually, I saw it is working on storybook. So I checked that the text size is applied by the `sizeMap[size]` and the Tailwind `tw-font-*` classes are to apply the `font-weight`, not `font-size`.
So, everything is ok and the logic is correct. My bad. :)